code/addons/onboarding/example-stories/button.css

```diff
@@ -0,0 +1,30 @@
+.storybook-button {
```

Comment:
I have a suspicion that these classnames might clash currently or accidentally in the future.

Would it be a lot of trouble to make these specific to the example?
As this isn't a CSS module AFAICS.